analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -122,6 +122,22 @@ class MavenSupport(private val workspaceReader: WorkspaceReader) {
             }
         }
 
+        fun parseOrganizations(mavenProject: MavenProject) =
+            sortedSetOf<String>().apply {
+                mavenProject.organization?.let {
+                    if (!it.name.isNullOrEmpty()) {
+                        add(it.name)
+                    }
+                }
+                mavenProject.developers.mapNotNull {
```

Comment:
The usage of `mapNotNull` is a bit strange here, as nothing is done with the result of the mapping. I assume, we want something like:

```
val authors = mavenProject.developers.mapNotNull {
  it.organization?.takeUnless { it.isEmpty() } ?: it.name
}
addAll(authors)

```
Or `authors` could even be inlined.